examples/psyir/Makefile

```diff
@@ -37,7 +37,7 @@ include ../common.mk
 
 transform:
 	${PYTHON} create.py
-	${PYTHON} create_aggregate_types.py
+	${PYTHON} create_structure_types.py
```

Comment:
I agree this is a better name, but there are a couple of references to it to update:
```
doc/user_guide/examples.rst
461:``create_aggregate_types.py`` demonstrates the representation of

examples/psyir/README.md
63:> python create_aggregate_types.py
```

Reply:
Oops. Done now.